modules/virt-collecting-data-about-your-environment.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+
+:_content-type: Procedure
+[id="virt-collecting-data-about-your-environment_{context}"]
+= Collecting data about your environment
+
+Collecting data about your environment minimizes the time required to analyze and determine the root cause.
+
+.Prerequisites
+
+* Set the retention time for Prometheus metrics data to a minimum of seven days.
+* Configure the Alertmanager to capture relevant alerts and send them to a dedicated mailbox so that they can be viewed and persisted outside the cluster.
+* Record the exact number of affected nodes and virtual machines.
+
+.Procedure
+
+. Collect `must-gather` data for {product-title}.
+. Collect `must-gather` data for {rh-storage-first}, if necessary.
+. Collect `must-gather` data for {VirtProductName}.
+. Collect Prometheus metrics for virtual resources.
```

Comment:
We need to collect all metrics of the cluster (not just virt)

Reply:
I updated the wording. However, the only doc that I can link to is [Querying metrics for all projects as a cluster administrator](https://docs.openshift.com/container-platform/4.11/monitoring/managing-metrics.html#querying-metrics-for-all-projects-as-an-administrator_managing-metrics).